tools/clang/test/HLSLFileCheck/hlsl/workgraph/invalid_barrier_1.ll

```diff
@@ -18,13 +18,13 @@ target triple = "dxil-ms-dx"
 define void @node01() #0 {
   
   ; CHECK: Invalid memory type flag  
-  call void @dx.op.barrierByMemoryType(i32 244, i32 16, i32 1, i32 1)  ; BarrierByMemoryType(MemoryTypeFlags,AccessFlags,SyncFlags)
+  call void @dx.op.barrierByMemoryType(i32 244, i32 16, i32 5)  ; BarrierByMemoryType(MemoryTypeFlags,AccessFlags,SemanticFlags)
```

Comment:
It will have no effect, but the comment isn't quite right. SyncFlags wasn't renamed, it was merged with Access Flags to create SemanticFlags.

Reply:
Oh yeah, that's what I mean to do but I did not delete the Access Flags :(.